cypress/e2e/users_spec/UsersCreation.cy.ts

```diff
@@ -66,122 +66,87 @@ describe("User Creation", () => {
-    userCreationPage.clickElementById("user-profile-name");
-    userCreationPage.clickElementById("profile-button");
-    userCreationPage.verifyElementContainsText(
-      "username-profile-details",
-      "devdistrictadmin",
-    );
-    userCreationPage.clickElementById("edit-cancel-profile-button");
-    userCreationPage.typeIntoElementByIdPostClear(
-      "firstName",
-      "District Editted",
-    );
-    userCreationPage.typeIntoElementByIdPostClear("lastName", "Cypress");
-    userCreationPage.selectDropdownOption("gender", "Male");
-    userCreationPage.typeIntoElementByIdPostClear("phoneNumber", phone_number);
-    userCreationPage.typeIntoElementByIdPostClear(
-      "altPhoneNumber",
-      emergency_phone_number,
-    );
-    userCreationPage.typeIntoElementByIdPostClear("email", "test@test.com");
-    userCreationPage.typeIntoElementByIdPostClear("weekly_working_hours", "14");
+    userCreationPage.clickProfileName();
+    userCreationPage.clickProfileButton();
+    cy.verifyContentPresence("#username-profile-details", ["devdistrictadmin"]);
+
+    userCreationPage.clickEditCancelProfileButton();
+    userCreationPage.typeIntoFirstNameAndClear();
+    userCreationPage.typeIntoLastNameAndClear();
+    cy.clickAndSelectOption("#gender", "Male");
+    userCreationPage.typeIntoPhoneNumberAndClear(phone_number);
+    userCreationPage.typeIntoAltPhoneumberAndClear(emergency_phone_number);
+    userCreationPage.typeIntoEmailAndClear();
+    userCreationPage.typeIntoWeeklyWorkingHoursAndClear();
     userCreationPage.typeIntoElementByIdPostClearDob(
       "date_of_birth",
       "01011998",
     );
-    userCreationPage.clickElementById("submit");
-    userCreationPage.verifyElementContainsText(
-      "contactno-profile-details",
+    cy.submitButton("Update");
+    cy.verifyContentPresence("#contactno-profile-details", [
       "+91" + phone_number,
-    );
-    userCreationPage.verifyElementContainsText(
-      "whatsapp-profile-details",
+    ]);
+    cy.verifyContentPresence("#whatsapp-profile-details", [
       "+91" + emergency_phone_number,
-    );
-    userCreationPage.verifyElementContainsText(
-      "firstname-profile-details",
+    ]);
+    cy.verifyContentPresence("#firstname-profile-details", [
       "District Editted",
-    );
-    userCreationPage.verifyElementContainsText(
-      "lastname-profile-details",
-      "Cypress",
-    );
-    userCreationPage.verifyElementContainsText(
-      "date_of_birth-profile-details",
-      "01/01/1998",
-    );
-    userCreationPage.verifyElementContainsText(
-      "emailid-profile-details",
-      "test@test.com",
-    );
-    userCreationPage.verifyElementContainsText(
-      "gender-profile-details",
-      "Male",
-    );
-    userCreationPage.verifyElementContainsText(
-      "averageworkinghour-profile-details",
-      "14",
-    );
+    ]);
+    cy.verifyContentPresence("#lastname-profile-details", ["Cypress"]);
```

Comment:
```suggestion
    cy.verifyContentPresence("#lastname-profile-details", [lastName]);
```

- move all the input values to a const
- then use the const for input as well as data verification purpose